src/Cmts.ml

```diff
@@ -270,20 +270,20 @@ let partition_after_prev_or_before_next t ~prev cmts ~next =
         let same_line_as_next l =
           next.loc_start.pos_lnum = l.loc_start.pos_lnum
         in
-        let colon_before =
-          let loc_end =
-            {prev.loc_end with pos_cnum= prev.loc_end.pos_cnum - 1}
-          in
+        let sequence_symbol_before =
+          let pos_cnum = prev.loc_end.pos_cnum - 1 in
+          let loc_end = {prev.loc_end with pos_cnum} in
           let char_loc = {prev with loc_start= loc_end} in
           let str = Source.string_at t.source char_loc in
-          String.equal str ";"
+          let c = str.[0] in
+          Char.equal c ';' || Char.equal c '|' || Char.equal c '&'
```

Comment:
Why not all infix operators ? `Ast.is_infix_id` do this.

Reply:
There would be a cyclic dependency between Ast and Cmts, I copied 2 lines of pattern matching from `Ast.is_infix_id` instead. Re-organizing all these utility functions into another file is out of the scope of this PR we can do it later.

Reply:
It does not handle operators like `land`. I think you should do this while you can otherwise we will forget and won't do it.

Reply:
done